src/With/Linq/LinqExtensions.cs

```diff
@@ -124,35 +39,12 @@ public static int FindIndex<T>(this IEnumerable<T> self, Func<T, bool> predicate
             return -1;
         }
 
-        public static bool None<T>(this IEnumerable<T> self, Func<T, bool> predicate)
-        {
-            return !Enumerable.Any(self, predicate);
-        }
-        public static bool None<T>(this IEnumerable<T> self)
-        {
-            return !Enumerable.Any(self);
-        }
-
-        public static bool One<T>(this IEnumerable<T> self, Func<T, bool> predicate)
-        {
-            return Enumerable.Count(self, predicate) == 1;
-        }
-        public static bool One<T>(this IEnumerable<T> self)
-        {
-            return Enumerable.Count(self) == 1;
-        }
-
         public static Partition<T> Partition<T>(this IEnumerable<T> self, Func<T, bool> partition)
```

Comment:
Is there a similar function in any other language (or library), and what is it called?

Reply:
In [f#](https://msdn.microsoft.com/en-us/visualfsharpdocs/conceptual/list.partition%5B't%5D-function-%5Bfsharp%5D), [Ruby](https://ruby-doc.org/core-2.5.1/Enumerable.html#method-i-partition) and [haskell](http://hackage.haskell.org/package/base-4.11.1.0/docs/Data-List.html#v:partition)